src/translations/fr.json

```diff
@@ -217,5 +217,15 @@
     "StopComputation": "Arrêter le calcul",
     "ColumnsList": "Liste des colonnes",
     "LabelSelectList": "Configurer colonnes",
-    "CheckAll": "Sélectionner tout / aucun"
+    "CheckAll": "Sélectionner tout / aucun",
+
+    "LockoutLine": "Consigner la ligne",
+    "TripLine": "Déclencher la ligne",
+    "EnergiseOnOneEnd": "Mettre sous tension à vide depuis {substation}",
+    "SwitchOnLine": "Fermer la ligne",
```

Comment:
Mettre en service ?

Reply:
Changed to "Mettre en service la ligne" as stated in the US.

Reply:
Change `UnableToSwitchOnLine` message also ?